src/converter/pytorch_converter.py

```diff
@@ -346,10 +349,17 @@ def get_protobuf_node_type_from_json_node(
                     if parent_node.name == "record_param_comms"
                     else parent_node.name
                 )
+                if parent_node.name == "record_param_comms" and parent_node.pg_name != "":
+                    json_node.pg_name = parent_node.pg_name
                 if "send" in keyword:
                     return COMM_SEND_NODE
                 if "recv" in keyword:
                     return COMM_RECV_NODE
+                # In NCCL, all-to-all communication is implemented using point-to-point
+                # communications. More details can be found here:
+                # https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/usage/p2p.html
+                if "nccl:all_to_all" in keyword:
```

Comment:
This string name `nccl::all_to_all` is hardcoded below:

https://github.com/pytorch/pytorch/blob/7c71ab1d40992ea2660bb124152e95b7a9a5119d/torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp#L4974

There is a possibility that once this changes in PT, this breaks.